src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,97 @@
+//SECTION 1:
+import React, { FC, useState } from 'react';
+import { classNames } from '../../utils/classNames';
+
+//SECTION 2:
+// DEWFINE OPTIONS AVAILABLE TO YOUR END USER
+
+export type SwitchSize = 'md' | 'sm';
+export type SwitchColor = 'default' | 'purple' | 'orange' | 'pink';
+
+//SECTION 3:
+// INTERFACE
+
+export interface ISwitchProps {
```

Comment:
Add descriptions for the Prop Types table on Storybook

Reply:
I finished